docs/admin_guide/cloud_role_mapping.rst

```diff
@@ -0,0 +1,95 @@
+
+==================
+AWS Role chaining
+==================
+
+Using an EC2 instance profile you can allow, your hopsworks cluster, access to AWS resources. This forces all hopsworks users to
+share the instance profile role and the resource access policies attached to that role. To allow for per project access policies 
+you could have your users use AWS credentials directly in their programs which is not recommended or you can and should use
```

Comment:
```suggestion
you could have your users use AWS credentials directly in their programs which is not recommended and should use
```
```suggestion
you could have your users use AWS credentials directly in their programs which is not recommended or you can and should use
```

Reply:
I do not think the sentence makes sense if I remove or